includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -797,6 +798,30 @@ public function is_valid_optimized_checkout_page(): bool {
 	}
 
+	/**
+	 * Whether Optimized Checkout layout should be used for this request.
+	 *
+	 * When false, split Payment Element path is used.
+	 *
+	 * @return bool
+	 */
+	public function should_use_optimized_checkout_payment_method_layout(): bool {
+		return $this->oc_enabled && $this->is_valid_optimized_checkout_page() && WC_Stripe_Helper::is_stripe_gateway_first_in_available_list( self::ID );
+	}
```

PHPMD (2.15.0)

[error] The method should_use_optimized_checkout_payment_method_layout is not named in camelCase. (undefined)

(CamelCaseMethodName)